### ISSUE TITLE
Should ARIA 19 be mapped to SC 1.3.1 or 4.1.2?

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2245]
Technique ARIA 19 is mapped to SC 3.3.1 (and 4.1.3) although it seems that it's more likely to map to SC 1.3.1 or SC 4.1.2 instead of 3.3.1.   SC 4.1.2 is scoped to user interface components which may not be applicable to non-interactive content which leaves SC 1.3.1 Information and Relationships.  This leads to a broader discussion on where live region issues that are not status messages should be mapped - 4.1.2 if user interface component or 1.3.1 if neither?

https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19

Trusted tester maps to 4.1.2.
https://section508coordinators.github.io/ICTTestingBaseline/05Changing.html

**Comment by patrickhlauke** [comment]
> This leads to a broader discussion on where live region issues that are not status messages should be mapped - 4.1.2 if user interface component or 1.3.1 if neither?

just on this part, I believe (I may be wrong though) that no SC directly covers non-status-message live region problems/misuses per se. which is why 4.1.3 was needed in the first place (but scoped very tightly to just status messages, as the topic of what is/isn't/should/shouldn't be a live region was too broad/subjective)

for what it's worth, since 4.1.3 came around, i've been known to write up best practice recommendations for these cases that reference 4.1.3 (but don't normatively fail 4.1.3)

**Comment by JAWS-test** [comment]
ARIA 19 is about error messages.

Defintion of status messages is:

>  change in content ... that provides information to the user .. on the existence of errors

That's why I don't understand why 4.1.3 shouldn't fit. 4.1.2, 1.3.1 and 3.3.1 do not apply in my opinion. This is because before WCAG 2.1 there was no requirement for error messages to be marked as a live region. An error message that is not a live region would not have been a problem with WCAG 2.0, it would have met 4.1.2, 1.3.1 and 3.3.1. That is why WCAG 2.1 then introduced 4.1.3.

**Comment by mraccess77** [comment]
@JAWS-test I agree 4.1.3 applies to ARIA 19.  I'm not sure ARIA 19 as the test procedure is written is sufficient to meet SC 3.3.1 although it's related to SC 3.3.1.

For SC 4.1.2 it says "and values that can be set by the user can be [programmatically set](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html#programmaticallysetdef); and notification of changes to these items is available to [user agents](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html#useragentdef)" which indicates at least for components changes could be covered by SC 4.1.2.

But I'm still not clear if a user interface component includes non-interactive content like text and if we count associated error messages as part of the component or not.

**Comment by awkawk** [comment]
@mraccess77 What are your concerns about meeting 3.3.1 with this technique?

**Comment by scottaohara** [comment]
the technique shows how one can add error messages, meeting 3.3.1.  That part is not specific to 'ARIA' but is rather tangential to the alert that does necessitate ARIA.

So, I'm not sure what the problem here is, unless - being an ARIA technique - you don't want to indicate that ARIA is necessary to meet 3.3.1, which it is not.

**Comment by mraccess77** [comment]
Regarding applicability to 3.3.1 - I would suggest the check in ARIA 19 be a little more specific around the error message e.g., "the item that is in error is identified and the error is described to the user in text".    But more generally the purpose of my ask for clarification was around 4.1.2 from the Trusted Tester mapping - and that has been discussed above.
